src/types.ts

```diff
@@ -0,0 +1,34 @@
+import { App } from 'vue'
+
+export interface DriftWidget {
+  /**
+   * Installs the drift widget into an app
+   */
+  install(
+    app: App,
+    widgetId: string,
+    config?: OptionalVueDriftWidgetConfig,
+  ): void
+}
+
+export interface VueDriftWidgetConfig {
+  /**
+   * Plugin can be disabled by setting this to `false`. Useful if you'd like to load the script but: `enabled: !!GDPR_Cookie`
```

Comment:
talking about gdpr this should be true for loading the script as this will set the 3rd party cookies not executing the load() function...

Reply:
I guess it should be set to false then to follow your opinion? Anyways, this should be optional, so will leave it like it is

Reply:
but the comment is misleading as `enable=false` will not help with GDPR..  only if the script is not loaded from 3rd party servers.

Reply:
nope that isn't right. The external sources will be loaded but only initialized when enabled = true. Before we call drift.load(), the environment will not be established and no tracking or cookies will be set. We can load the sources but not enable the tracking. This is not an GDRP issue.